src/app/components/WinModal.ts

```diff
@@ -0,0 +1,28 @@
+import { Player } from '~src/Player';
+import { Render } from '../utils/Render';
+import { ModalBasic } from './ModalBasic';
+
+export class WinModal extends ModalBasic {
+  create(player: Player): void {
+    this.renderBasicModal(
+      'CONGRATULATIONS!',
+      `${player.theName} has won!`,
+      this.createImage(player),
+    ) as WinModal;
+  }
+
+  addButton(): void {
```

Comment:
Perhaps addEndGameButton? It has the text 'End game - back to menu' inside, so perhaps the name of the method could be clearer